hca/util/__init__.py

```diff
@@ -480,11 +485,64 @@ def build_argparse_subparsers(self, subparsers):
             sig = signature(command)
             if not getattr(command, "__doc__", None):
                 raise SwaggerClientInternalError("Command {} has no docstring".format(command))
-            doc = command.__doc__.strip().format(prog=subparsers._prog_prefix)
-            command_subparser = subparsers.add_parser(command.__name__,
-                                                      help=doc.splitlines()[0],
-                                                      description=doc)
+            docstring = command.__doc__.format(prog=subparsers._prog_prefix)
+            method_args = self.parse_docstring(docstring)
+            command_subparser = subparsers.add_parser(command.__name__.replace("_", "-"),
+                                                      help=method_args['summary'],
+                                                      description=method_args['description']
+                                                      )
             command_subparser.set_defaults(entry_point=self._command_arg_forwarder_factory(command, sig))
             for param_name, param_data in sig.parameters.items():
                 command_subparser.add_argument("--" + param_name.replace("_", "-"),
+                                               help=method_args['params'].get(param_name, None),
                                                **self._get_command_arg_settings(param_data))
+
+    @staticmethod
+    def parse_docstring(docstring):
```

Comment:
To make package organization more consistent and uncrowd this file, please detach this method and move it to `hca/util/_docs.py`.

Reply:
moved to hca/util/_docs.py